src/plugins/footnotes/index.ts

```diff
@@ -144,7 +153,8 @@ const deleteFootnoteWidget =
 
         // delete table footnotes
```

Comment:
Since this code is also used for deleting regular footnotes, let's  remove the word "table".